src/components/ImageWithSizeCalculation.js

```diff
@@ -1,10 +1,13 @@
 import React, {PureComponent} from 'react';
 import {View, Image} from 'react-native';
```

Comment:
@Beamanator can you use `Image` from `react-native-fast-image` 

```js
import Image from '@pieter-pot/react-native-fast-image';
```

Or does that break a lot of stuff?

Perhaps headers would be fixed on Android this way?

Reply:
Oohhh I will definitely try that tomorrow